src/wstool/cli_common.py

```diff
@@ -172,7 +172,7 @@ def _get_status_flags(basepath, elt_dict):
     return mflag
 
 
-def get_info_table_elements(basepath, entries):
+def get_info_table_elements(basepath, entries, unmanaged):
```

Comment:
since this is an api change, would be nicer to provide a default for the new additional parameter.